pkgs/by-name/go/go-enum/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "go-enum";
+  version = "0.9.1";
+
+  src = fetchFromGitHub {
+    owner = "abice";
+    repo = "go-enum";
+    rev = "v${version}";
+    hash = "sha256-1eAMCr7GeXwzHS1E9Udp0l2eMOiihhm7aAOQEyKNQa8=";
+  };
+
+  vendorHash = "sha256-AlJzwJtQaJNqulw9alltwSw8gVEBx58cejlkgXYuOAI=";
+
+  meta = with lib; {
```

nit: don't bring lib into scope. https://github.com/NixOS/nixpkgs/issues/208242

```suggestion
  meta = {
```